src/renderer/features/assets/AssetsPortfolioView/model/portfolio-model.ts

```diff
@@ -105,22 +108,25 @@ sample({
   },
   fn: ({ connections, chains, tokens, activeWallet }): AssetByChains[] => {
     const isMultisigWallet = walletUtils.isMultisig(activeWallet);
+    const multisigChainToInclude = isMultisigWallet ? activeWallet.accounts[0].chainId : undefined;
```

Comment:
Plz check account before diving into chainId

Reply:
<img width="612" alt="image" src="https://github.com/user-attachments/assets/a555590d-d343-49e1-99b7-f784b36bac21">
It's already `MsigAccount`, or you mean check index 0?

Reply:
index